config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-periodics.yaml

```diff
@@ -0,0 +1,30 @@
+periodics:
+- name: periodic-cluster-api-provider-azure-conformance-v1alpha3
+  decorate: true
+  decoration_config:
+    timeout: 3h
+  interval: 12h
+  labels:
+    preset-dind-enabled: "true"
+    preset-kind-volume-mounts: "true"
+    preset-azure-cred: "true"
+  extra_refs:
+    - org: kubernetes-sigs
+      repo: cluster-api-provider-azure
+      base_ref: master
+      path_alias: "sigs.k8s.io/cluster-api-provider-azure"
+  spec:
+    containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20200407-c818676-experimental
```

Comment:
```suggestion
      - image: gcr.io/k8s-testimages/kubekins-e2e:v20200407-c818676-master
```

Reply:
I noticed all the other Azure jobs have experimental which is why I set this one to experimental as well, do we need to change those? Not sure what the difference is between the experimental and master image